package.json

```diff
@@ -32,8 +32,8 @@
     "grunt-contrib-clean": "^1.0.0",
     "grunt-contrib-jshint": "^1.0.0",
     "grunt-contrib-nodeunit": "^1.0.0",
-    "loopback": "^1.7.4",
-    "loopback-datasource-juggler": "^1.3.10"
+    "loopback": "^2.0.0",
```

Comment:
@bajtos Changed the version because of this [issue](https://github.com/strongloop/loopback-sdk-angular/issues/236)